src/test/java/org/springframework/data/couchbase/domain/AbstractEntity.java

```diff
@@ -0,0 +1,75 @@
+package org.springframework.data.couchbase.domain;
```

Comment:
license header missing

Reply:
I've added the license header to all the files that were missing it.